lib/Backend/GlobOpt.cpp

```diff
@@ -6935,20 +6959,34 @@ GlobOpt::SetSymStore(ValueInfo *valueInfo, Sym *sym)
     }
     if (valueInfo->GetSymStore() == nullptr || valueInfo->GetSymStore()->IsPropertySym())
     {
-        valueInfo->SetSymStore(sym);
+        SetSymStoreDirect(valueInfo, sym);
     }
 
     return sym;
 }
 
 void
+GlobOpt::SetSymStoreDirect(ValueInfo * valueInfo, Sym * sym)
```

Comment:
Why Direct instead of just SetSymStore?  The direct part made me believe at first that this version didn't set the changed bit.